README.md

```diff
@@ -242,7 +242,6 @@
   |> String.slice(1, 3)
   ```
 
-  <a name="needless-pipeline"></a>
```

Comment:
It is, unfortunately, very frequent issue, thus this link was added on purpose to be able to point to this section directly. :bowtie: